tv_apps/smart-system/js/app_install_manager.js

```diff
@@ -121,6 +132,9 @@ var AppInstallManager = {
   handleHomeButtonPressed: function ai_handleHomeButtonPressed(e) {
     this.dialog.classList.remove('visible');
```

Comment:
The code here looks not good. We should maintain a variable to keep the shown dialog.  and only hide it while press home. In this function, we always calls handleXXXCancel on install or uninstall dialog.